tests/units/anta_tests/test_interfaces.py

```diff
@@ -1190,7 +1190,10 @@
         },
         "expected": {
             "result": "failure",
-            "messages": ["The following interface(s) are not in the expected state: ['Ethernet2 is up/unknown'"],
+            "messages": [
+                "Interface status/Line protocol status for Ethernet2 - Expected: up/down, Actual: up/unknown",
+                "Interface status/Line protocol status for Ethernet8 - Expected: up/up, Actual: up/down",
```

Comment:
for eth8 we have not given the lineProtocolStatus as input so why it is validating and having it in failure message